### ISSUE TITLE
Allow dropping unreachable tables

### ISSUE THREAD

**Issue by timmaxw** [rethinkdb/rethinkdb#4335]
Currently an attempt to drop a completely unreachable table fails with:
```
The server(s) hosting table `test.total_loss` are currently unreachable.
The table was not dropped.
```

**Comment by timmaxw** [comment]
In CR 2961.

**Comment by timmaxw** [comment]
Fixed in 05e7170c1b8a942630a07880e4cfa71c7dc8cb6b.
